src/xpcc/architecture/platform/driver/core/cortex/stm32/stm32_idtcm.ld.in

```diff
@@ -486,6 +486,41 @@ SECTIONS
 		__table_copy_extern_end = .;
 	} >FLASH
 
+%% if parameters.allocator == "tlsf"
+	/* SRAM properties bit mask (16-bit):
+	 *
+	 * - 0x0001: accessible via S-Bus
+	 * - 0x0002: accessible via D-Bus
+	 * - 0x0004: accessible via I-Bus
+	 * - 0x0008: accessible via DMA
+	 * - 0x0010: accessible via DMA2D
+	 *
+	 * - 0x1FE0: reserved
+	 *
+	 * - 0x2000: Fast memory (Core- or Tightly-Coupled)
+	 * - 0x4000: non-volatile (or battery backed) memory
+	 * - 0x8000: external memory
+	 */
+	.table.heap : ALIGN(4)
+	{
+		__table_heap_start = .;
+		LONG (0x001f) /* SRAM1 */
+		LONG (__heap1_start)
+		LONG (__heap1_end)
+		LONG (0x001f) /* SRAM2 */
+		LONG (__heap2_start)
+		LONG (__heap2_end)
+		LONG (0x200f) /* DTCM */
+		LONG (__heap0_start)
+		LONG (__heap0_end)
+		LONG (0x2004) /* ITCM */
```

Comment:
Confirm that the ITCM cannot be accessed via D-Bus or similar.
How would you get the instruction data in there in the first place.
If it were not accessible by D-Bus, then the allocator pool initialization in this region will fail.